examples/models/llama/model.py

```diff
@@ -259,7 +259,7 @@ def get_example_inputs_kvcache_sdpa(self):
         else:
             return (
                 torch.tensor(
-                    [[1]], dtype=torch.long
+                    [[1]], dtype=torch.int32
```

Comment:
Can we put it in model metadata and get it in runtime to avoid misuse

Reply:
Oops, I miss this change. Yes, I just would like to change it with QNN Backend.